app/controllers/coronavirus_form/able_to_leave_controller.rb

```diff
@@ -33,6 +33,6 @@ def group
   end
 
   def previous_path
-    "/"
+    previous_question(controller_name)
```

Comment:
If this is always the same, it can probably be moved the application controller.